djangocms_bootstrap4/contrib/bootstrap4_grid/templates/djangocms_bootstrap4/admin/grid_row.html

```diff
@@ -0,0 +1,13 @@
+{% extends "djangocms_bootstrap4/admin/base.html" %}
+{% load staticfiles %}
+
+{% block extrahead %}
+    {{ block.super }}
+    <script src="{% static 'djangocms_bootstrap4/js/bundle.grid.js' %}"></script>
+{% endblock %}
+
+{% block field_sets %}
+    <div class="djangocms-bootstrap4 djangocms-bootstrap4-row" data-static="{% static "" %}">
```

Comment:
huh?

Reply:
I need the static path for configuration

Reply:
{% get_static_prefix %}